src/viewEngine.js

```diff
@@ -45,11 +46,25 @@ function registerTargets(group, prefix, dir) {
     if (!fs.existsSync(`${__dirname}/tmp/`)) {
       fs.mkdirSync(`${__dirname}/tmp/`);
     }
-    fs.writeFileSync(mountTarget, `
-      var mount = require('${__dirname}/common/frontend/helpers/page').mount;
-      var Page = require('${file}').default;
-      mount(Page);
-    `);
+
+    var script = `
+      const mount = require('${__dirname}/browser/renderer').mount;
```

Comment:
Okay, so here we're basically autogenerating code to "mount" pages. We do a server side rendering, but then those aren't real javascript components that can actually update until React runs on the page.  So mounting just re-renders the component client side so that it's a live, active component.

This is here just because for every view we have, we always mount it, and it was a lot of redundant code. This autogenerates a mount script and saves it in a temporary directory so that mounting of components stays separate from the components themselves.